syne_tune/search_space.py

```diff
@@ -595,10 +595,11 @@ class FiniteRange(Domain):
     """
     Represents a finite range `[lower, ..., upper]` with `size` values
     equally spaced in linear or log domain.
+    If `is_int`, the value type is int (rounding after the transform).
 
     """
     def __init__(self, lower: float, upper: float, size: int,
-                 log_scale: bool = False):
+                 log_scale: bool = False, is_int: bool = False):
```

Comment:
`cast_int` would be a better name since we cast ultimately to differences that may not be too small.